### ISSUE TITLE
[Bug] Quick Install App PrestaShop Error: "install" directory is missing

### ISSUE THREAD

**Issue by lapriente** [hestiacp/hestiacp#2644]
### Describe the bug

Tested on Debian 10, 11
After installing PrestaShop with Quick Install App instead of the main page, I see:
Error: "install" directory is missing

### Tell us how to replicate the bug

1. Click on the web tab
2. Add domain
3. Enable Let's Encrypt
4. Install Presta Shop through Quick Install App
5. Open main page

### Which components are affected by this bug?

Control Panel Web Interface

### Hestia Control Panel Version

1.5.15

### Operating system

Debian 11.3

### Log capture

```shell
2022-06-05 12:29:11 v-run-cli-cmd  'admin' 'ls' '/home/admin/web/localhost/public_html/'
2022-06-05 12:29:21 v-add-database  'admin' '66853' '66853' '******'
2022-06-05 12:29:35 v-run-cli-cmd  'admin' 'ls' '/home/admin/web/localhost/public_html/'
```

**Comment by ScIT-Raphael** [comment]
Do not run any website within the admin user, create a new user and retry your test.

**Comment by jaapmarcus** [comment]
It is an know issue is patched in Beta 1.6.0
